model/tactile_markers.urdf.xacro

```diff
@@ -42,6 +42,10 @@
 			<parent link="${prefix}${finger}_middle_link"/>
 			<tactile channel="${channel}">
 				<xacro:taxel mesh="${F}mo" idx="${tactMap.get(F+'mo', -1)}"/>
+				<!--tmm index does not exist for L1 thumb but tmo+tmm meshes cover the tmo L1 patch -->
+				<xacro:if value="${layout=='L1' and finger=='thumb'}">
+					<xacro:taxel mesh="${F}mm" idx="${tactMap.get(F+'mo', -1)}"/>
+				</xacro:if>
 				<xacro:taxel mesh="${F}mm" idx="${tactMap.get(F+'mm', -1)}"/>
```

Comment:
@guihomework: What do you think about this one:
I want to avoid having mm listed twice. This way it becomes clear that mm is used but refers to two different indices.
```suggestion
				<!--tmm index does not exist for L1 thumb but tmo+tmm meshes cover the tmo L1 patch -->
				<xacro:if value="${layout=='L1' and finger=='thumb'}">
					<xacro:taxel mesh="${F}mm" idx="${tactMap.get(F+'mo', -1)}"/>
				</xacro:if>
				<xacro:taxel mesh="${F}mm" idx="${tactMap.get(F+'mo', -1) if (layout=='L1' and finger=='thumb') else tactMap.get(F+'mm', -1)}"/>
```

Reply:
I prefer this version to clearly see the special case. (but the suggestion still includes the xacro:if somehow which is then duplicate, I suppose the change will be done separately anyway, one cannot "commit suggestion" as is)

What about the other changes done in #4, where other taxels are also listed twice (ptmp ptop both connected to ptop for instance). To be consistent with you wish to not have them twice, one would need to change it the same way.

I suggest to not merge this there, but fix the previous PR first with these xacro changes and keep the taxel center fix separate.